src/api/dependency-discovery/README.md

```diff
@@ -28,7 +28,17 @@ By default, the server is listening on port 10500 (http://localhost:10500).
 
 ### Docker
 
-TODO
+To run the service with only the container, you may use the following command:
+
+```bash
+docker run -p 10500:10500 telescope_dependency_discovery_svc:latest
```

Comment:
Bugging you one more time, can you explain this `10500:10500`?

Reply:
This means that the service running inside the container uses port `10500`, and we want to bind port `10500` in the host to it so we can access the service from our local machine.  The `-p` means `publish` and refers to the ports.  So you read it `-p host-port:container-port`

Reply:
Right!